pkg/bindings/containers/checkpoint.go

```diff
@@ -79,7 +79,13 @@ func Restore(ctx context.Context, nameOrID string, options *RestoreOptions) (*en
 
 	// Open the to-be-imported archive if needed.
 	var r io.Reader
-	if i := options.GetImportArchive(); i != "" {
+	i := options.GetImportArchive()
+	if i == "" {
+		// backwards compat, ImportAchive is a typo but we still have to
+		// support this to avoid breaking users
+		i = options.GetImportAchive()
```

Comment:
Can you add a `// TODO: remove with 5.0`?

Reply:
done